common/projectconfig/projectconfig.go

```diff
@@ -34,6 +34,15 @@ type Config struct {
 	FTLMinVersion string                      `toml:"ftl-min-version"`
 }
 
+// MustGetModuleDirs returns the module-dirs field from the ftl-project.toml, unless that
+// is not defined, in which case it defaults to the root directory.
+func (c Config) MustGetModuleDirs() []string {
```

Comment:
"MustX" is a pattern in Go that means "must get this thing or panic on error", so can we maybe call this "ModuleDirsOrDefault()" instead?

Reply:
Whoops! Thanks thanks, that's way better